mobile.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins&display=swap">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio Project</title>
+  </head>
+<body>
+    <div class="main-container">
+    <header>
+      <nav class="toolbar">
+          <img class="menu" src="images/ic_menu.png">
+          <img class="menu-shape" src="images/image_geometry_1.png">
+      </nav>
+      <section class="headline">
+        <ul>
+          <li class="my-name">Juan<br>Díaz</li>
+          <li class="myname-intro">Hello! I am a software developer! I can<br>
+                                 help you build a product, feature or<br> 
+                                 website. Take a look at my work.<br><br>      
+                                 If you like what you see and have a<br>
+                                 project you need coded, don’t hesitate to<br>
+                                 contact me.</li>
+        </ul>
+      </section>
+      <footer>
+        <button type="button" class="collaboration-box">Start collaboration</button>
+      </footer>
```

Comment:
- [ ] It would be better to remove this *footer* tag as this is a continuous project, we have not reached the footer of our page so remove this tag 👍